sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallMediaAsyncUnitTests.java

```diff
@@ -70,7 +68,7 @@ public void cancelAllOperationsWithResponse() {
 
     @Test
     public void recognizeWithResponse() {
-        RecognizeOptions recognizeOptions = new RecognizeOptions(RecognizeInputType.DTMF, new RecognizeConfigurations());
+        CallMediaRecognizeDtmfOptions recognizeOptions = new CallMediaRecognizeDtmfOptions(new CommunicationUserIdentifier("id"));
```

Comment:
Could you please add a test with a filled DtmfOptions object?
For me I was lacking this test that you have and it had errors, so, maybe we can discover or at least cover all bases having both tests :)